tests/generation/test_flax_utils.py

```diff
@@ -101,6 +101,10 @@ def test_greedy_generate_pt_fx(self):
             pt_model = pt_model_class(config).eval()
             pt_model = load_flax_weights_in_pytorch_model(pt_model, flax_model.params)
 
+            # Generate max 5 tokens only otherwise seems to be numerical error accumulation
+            pt_model.generation_config.max_length = 5
+            flax_model.generation_config.max_length = 5
```

Comment:
should it be `input_ids.shape[-1] + 5` or using `max_new_tokens` instead?

Reply:
yeah, could also be max-new-tokens. Interestingly after the linked PR `max_length` behaves same way as `max_new_tokens`